conformance/src/main/scala/za/co/absa/enceladus/conformance/CmdConfig.scala

```diff
@@ -77,8 +77,8 @@ object CmdConfig {
           case _    => success
         })
 
-    opt[Int]('r', "report-version").action((value, config) =>
-      config.copy(reportVersion = value)).text("Report version")
+    opt[Int]('r', "report-version").optional().action((value, config) =>
```

Comment:
Add to `.text` that if not provided it is inferred from previously run conformance.